_topic_maps/_topic_map.yml

```diff
@@ -840,6 +840,18 @@ Topics:
     File: file-integrity-operator-advanced-usage
   - Name: Troubleshooting the File Integrity Operator
     File: file-integrity-operator-troubleshooting
+- Name: cert-manager Operator for Red Hat OpenShift
```

Comment:
@slaskawi These docs will show for OKD too - just making sure that's okay?

Reply:
I'm not sure. For the time-being, I think they should be OCP only.

Reply:
Actually I'm wondering whether the Operator name having "for Red Hat OpenShift" would be a problem for OKD? Perhaps @Anandnatraj can weigh in on this too?

Reply:
Sorry @slaskawi I missed your reply before I commented again. I'll update it to be for OCP only, but let me know if we should allow it for OKD too.